### PR TITLE
Added SmartSDK NodeJS/JavaScript SDK for NGSIv2 to code_contributions.md

### PR DESCRIPTION
**Pull request by chicco785**
SmartSDK team developed and [SDK Client](https://github.com/smartsdk/ngsi-sdk-javascript) compatible with NSGIv2. 
It allows to develop both NodeJS applicaitions and JavaScript applications.

### CODE REVIEW

File: doc/manuals/code_contributions.md [hunk 1]
```diff
@@ -51,6 +51,59 @@ Thanks to Alejandro Villamarin (published around October 2013):
           System.err.println("Failed. Reason is " + e.getMessage());
        }
 
+## JavaScript and NodeJS
+SmartSDK team developed and [SDK Client](https://github.com/smartsdk/ngsi-sdk-javascript) compatible with NSGIv2.
```
Comment (fgalan) [inline doc/manuals/code_contributions.md]: A reference to the time in which code contribution has been produced should be included. E.g. (from other contributions in this document): "(published around June 2014)".

File: doc/manuals/code_contributions.md [hunk 2]
```diff
@@ -51,6 +51,59 @@ Thanks to Alejandro Villamarin (published around October 2013):
...
+
+NodeJS:
+```javascript
+var NgsiV2 = require('ngsi_v2');
+
+var defaultClient = NgsiV2.ApiClient.instance;
+
+// Configure API key authorization: fiware_token
+var fiware_token = defaultClient.authentications['fiware_token'];
+fiware_token.apiKey = "YOUR API KEY"
```
Comment (fgalan) [inline doc/manuals/code_contributions.md]: It is a good practise in JavaScript to include `;` at the end of the statements. In fact, you are following that criteria in most of your statement, but the following ones doesn't have it:

```
fiware_token.apiKey = "YOUR API KEY"
//fiware_token.apiKeyPrefix['X-Auth-Token'] = "Token"
var api = new NgsiV2.APIEntryPointApi()
```

The same in the client JavaScript fragment below.

File: doc/manuals/code_contributions.md (hunk 1 above)
Comment (chicco785) [inline doc/manuals/code_contributions.md]: Publication date has been added. Javascript syntax has been fixed.

**Comment by fgalan** [comment]
Thanks for the contribution!

LGTM

**Pull request merged**